main/src/cgeo/geocaching/ui/renderer/OfflineImageHandler.java

```diff
@@ -0,0 +1,31 @@
+package cgeo.geocaching.ui.renderer;
+
+import cgeo.geocaching.network.HtmlImage;
+
+import android.graphics.drawable.Drawable;
+import android.text.Spannable;
+import android.text.SpannableStringBuilder;
+import android.text.style.ImageSpan;
+
+import net.nightwhistler.htmlspanner.TagNodeHandler;
+import org.htmlcleaner.TagNode;
+
+public class OfflineImageHandler extends TagNodeHandler {
```

Comment:
Offline image handling weas not included in the solution before, or was it?

Reply:
It was. Would have been quite bad if images were always re-downloaded when opening the listing.

Here (the `HtmlImage` part is inlined):
https://github.com/cgeo/cgeo/blob/b3b521ca57542ca560341f616ba55d5cf1460d5e/main/src/cgeo/geocaching/CacheDetailActivity.java#L1877